### ISSUE TITLE
Onboarding Checklist for ES New Editor: Riva Quiroga

### ISSUE THREAD

**Issue by jenniferisasi** [programminghistorian/jekyll#1190]
This is an individual onboarding editor checklist for Rica Quiroga (@rivaquiroga), initially found [here](https://github.com/programminghistorian/jekyll/wiki/Onboarding-Checklist-for-New-Editors). 

We've put together this information to help you get up to speed. It includes details of what we will do and a checklist of tasks you should complete, to make sure you know how to perform key tasks. 

## What we will do
Within 2 weeks of your joining, our current team will ensure that you have been granted access to the following services used by the Project Team:

- [x] Our Google Group email so you can communicate with other editors (@acrymble)
- [x] Our Google Analytics account for monitoring web traffic (@acrymble)
- [x] Our Twitter team (@jenniferisasi )
- [x] Our Github Repository so you can upload materials to the site (@mdlincoln)
- [x] Our Skype group so that you can participate in our monthly calls (@amsichani)
- [x] We will also write a blog post welcoming you publicly to the team.
- [ ] partner you with an existing editor to 'shadow' the editing process so you can get a feel for it.

## What you should do
There are a few tasks you will need to perform to make sure you know how to use our system, and also to make sure your details appear on the Project Team page. All of these tasks are encouraged, but some may be optional:

### communication
- [x] set up a GitHub account and inform @mdlincoln of your username (mandatory)
- [x] set up a twitter account (optional)
- [x] set up a Skype account and inform @amsichani (mandatory)
- [x] inform @acrymble of your Google-activated email (mandatory)
- [x] write an email to the other editors using the Google group (say hello!) (mandatory)

### technical
We rely heavily on Github for communicating and conducting our editorial processes. You will need to get familiar with it and the way we use it. We recognise that this can take time, so if you would like one-on-one training, please let someone know and this will be arranged. To make sure you can do all required tasks, try the following activities on our Github /jekyll website.

- [x] contribute to one open ticket on our github repository (mandatory)
- [x] create and assign to yourself a new ticket on the github repository to ensure you know how to do this. Add a relevant label. (mandatory)
- [x] make a technical contribution on your own. This will normally be adding your bio to the /data/ph_authors.yml file, and uploading a square black and white avatar (.png) of yourself to /avatars). You will then submit a pull request for approval so that this bio information can be added to the Project Team page (mandatory)
- [x] Read all sections of the /contribute pages of the website, including author, reviewer, and editor guidelines
- [x] Bookmark the /jekyll (https://github.com/programminghistorian/jekyll) and /submissions (https://github.com/programminghistorian/ph-submissions) github sites so you can find them easily
- [x] Read the 'Making Technical Contributions' wiki page: https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions

### The Culture of the Project
We have invited you to join the team because we value your opinions. We encourage all of our editors to contribute to discussions on github or via email, and to civilly express their views, even if they disagree with any or all other members. This right to express one's opinions is central to our project. Please take this to heart.

**Comment by rivaquiroga** [comment]
Hi everyone!
Thank you very much for the onboarding. Here is the information required:

- @mdlincoln: my GitHub user is @rivaquiroga. 
- @amsichani: my Skype account is @rivaquiroga. 
- @acrymble: my Twitter handle is @rivaquiroga.
(bassically, I have the same username everywhere :nerd_face:) 

Thank you again! I'm very happy of being part of this awesome project.

**Comment by mdlincoln** [comment]
Thank you @rivaquiroga I've just sent you invitations to the github repository. It's great to have you on board 😸

**Comment by mariajoafana** [comment]
Bienvenida Riva!

**Comment by acrymble** [comment]
@rivaquiroga can you send me your email address (adam.crymble@gmail.com)

**Comment by jamotilla** [comment]
Bienvenida @rivaquiroga !!

**Comment by acrymble** [comment]
@jenniferisasi are you going to write the welcome blog post for our new members? It could be done in Spanish.

**Comment by jenniferisasi** [comment]
Yes, @acrymble, I have it almost ready. Waiting on them to upload the picture and profile to make it all connected, however, even if they don't, I will publish it on Friday.

**Comment by rivaquiroga** [comment]
@jenniferisasi, is there anything else you need me to do? Where do I have to upload my profile?

**Comment by jenniferisasi** [comment]
Hi @rivaquiroga. Our profiles are in [/data/ph_authors.yml](https://github.com/programminghistorian/jekyll/blob/gh-pages/_data/ph_authors.yml) You can follow my profile in the file as an example and can ask for French translation in your pull request. The image, as a .png goes in  [/avatars](https://github.com/programminghistorian/jekyll/tree/gh-pages/avatars). You can check our [Make Technical Contributions ](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions) for assistance, but let me know if you have any question.

**Comment by rivaquiroga** [comment]
@jenniferisasi,  picture and profile uploaded in #1230 :rocket: 
Thanks!

**Comment by acrymble** [comment]
@rivaquiroga is currently working with me to publish a lesson in progress, so I will close this on that basis, again noting that we hope to get her more actively editing soon.
